### PR TITLE
Fix double logo footer bug

### PR DESCRIPTION
**Pull request by eleanorreem**
See https://www.notion.so/chayn/Double-Bumble-logo-in-footer-e00b0c7de4df4b37a73621d0cff22771

### CODE REVIEW

File: components/layout/Footer.tsx [hunk 1]
```diff
@@ -102,7 +104,7 @@ const Footer = () => {
             partner.name === 'public' ? SOCIAL_LINK_CLICKED : PARTNER_SOCIAL_LINK_CLICKED;
 
           return (
-            <Box pr="2em" key={`${partner.name}_footer`}>
+            <Box pr="10px" key={`${partner.name}_footer`}>
```
Comment (swetha-chayn) [inline components/layout/Footer.tsx:107]: How come the spacing is in pixels? I thought we were moving towards ems

Comment (eleanorreem) [reply components/layout/Footer.tsx:107]: Ah poo. This was my bad. I pressed command Z too much and actually changed too much. 🤦‍♀️

Comment (eleanorreem) [reply components/layout/Footer.tsx:107]: Sorry should have double checked this commit!

File: components/layout/Footer.tsx [hunk 2]
```diff
@@ -94,6 +94,8 @@ const Footer = () => {
     setPartners(partnersList);
   }, [partnerAccesses, user, router]);
 
+  console.log(partners);
```
Comment (eleanorreem) [inline components/layout/Footer.tsx:97]: Rogue console log aswell tut

Comment (swetha-chayn) [reply components/layout/Footer.tsx]: lol

**Review by swetha-chayn — APPROVED** [review]

**Pull request merged**